lib/slack-ruby-bot-server/api/presenters/team_presenter.rb

```diff
@@ -10,6 +10,7 @@ module TeamPresenter
         property :team_id, type: String, desc: 'Slack team ID.'
         property :name, type: String, desc: 'Team name.'
         property :domain, type: String, desc: 'Team domain.'
+        property :state, type: String, desc: 'State parameter returned from Add to Slack response.'
```

Comment:
State is unlikely to be used other than to verify something once, lets not add it to the model.